src/test/resources/quick/unknown_namespace_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sys:overwritePolicy": "PRAGMATIC",
+  "{http://www.unknown}name": "ignored",
+  "{http://www.alfresco.org/model/content/1.0}author": "Used"
+}
```

Comment:
New line at end of file.

Reply:
Deliberate. This is exactly what the T-Engine would send.